Linter Plugin installation Error

The following error occurs during installation of the linter plugin:
```
Installing https://luarocks.org/luafilesystem-1.8.0-1.src.rock
src/lfs.c:84:10: fatal error: lua.h: No such file or directory
 #include <lua.h>
          ^~~~~~~
compilation terminated.

 

Error: Build error: Failed compiling object src/lfs.o
gcc -O2 -fPIC -I/usr/include/lua5.3 -c src/lfs.c -o src/lfs.o
```

liblua5.3-dev is missing but liblua5.1-dev is installed (I have no idea why)
possible fix: installing liblua5.3-dev

--- a/src/plugins/analysis/linter/install.sh
+++ b/src/plugins/analysis/linter/install.sh
@@ -9,10 +9,7 @@ then
 	sudo dnf install -y ShellCheck || exit 1
 
 	echo " Installing lua linter..."
-	sudo dnf install -y lua || exit 1
-	sudo dnf install -y lua-devel || exit 1
-	sudo dnf install -y luarocks || exit 1
-
+	sudo dnf install -y lua lua-devel luarocks || exit 1
 	sudo luarocks install luafilesystem || exit 1
 	sudo luarocks install argparse || exit 1
 	sudo luarocks install luacheck || exit 1
@@ -29,7 +26,7 @@ else
 	sudo apt-get install -y shellcheck || exit 1
 
 	echo " Installing lua linter..."
-	sudo apt-get install -y luarocks || exit 1
+	sudo apt-get install -y luarocks lua5.3 liblua5.3-dev || exit 1
 	sudo luarocks install luafilesystem || exit 1
 	sudo luarocks install argparse || exit 1
 	sudo luarocks install luacheck || exit 1

